src/components/Dialog-Window/Audio/DialogAudio.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import * as React from "react";
+import { DialogAudio } from "./DialogAudio";
+
+export default {
+  title: "Molecules/Dialog Content/Dialog Audio",
+  component: DialogAudio,
+} as ComponentMeta<typeof DialogAudio>;
+
+const Template: ComponentStory<typeof DialogAudio> = args => (
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  <DialogAudio {...args} />
+);
+
+export const NoteSimple = Template.bind({});
+
+NoteSimple.args = {
+  audioTrack: { path: "https://bigsoundbank.com/UPLOAD/mp3/0001.mp3" },
+  description: "Lyd: Opplesning av en tale fra den franske revolusjonen 1789",
+  subText: (
+    <>
+      <h4>
+        Den franske revolusjon var en periode med store sosiale og politiske
+        omveltningene i Frankrike i perioden 1789-1799. Året 1789 markerer det
+        første viktige vendepunktet under revolusjonen. 14. juli dette året brøt
+        det ut masseopprør i Paris og fengselet Bastillen ble stormet.
+      </h4>
```

Comment:
Not strictly a flaw that the users will notice, but we should use the `<b>` element here. It's important also for documentation to follow accessibility guidelines, because someone might see this example and copy it.

The reason why we're not using `<strong>` instead of `<b>` is that `<strong>` is used for accessibility purposes (screen readers will shout), while `<b>` is only used for making text look bold.

```suggestion
      <b>
        Den franske revolusjon var en periode med store sosiale og politiske
        omveltningene i Frankrike i perioden 1789-1799. Året 1789 markerer det
        første viktige vendepunktet under revolusjonen. 14. juli dette året brøt
        det ut masseopprør i Paris og fengselet Bastillen ble stormet.
      </b>
```